test/testutil/driver.c

```diff
@@ -257,14 +255,16 @@ PRINTF_FORMAT(2, 3) static void test_verdict(int verdict,
     test_flush_stdout();
     test_flush_stderr();
 
+    if (verdict == 0 && seed != 0)
```

Comment:
Why not just always print the seed.

Reply:
It's noise unless there is a failure.  I also need to catch the case where the seed hasn't been set because it isn't used then.